Solitaire/js/main.js

```diff
@@ -1,4 +1,259 @@
-
+(function () {
+    var $btnReset = document.getElementById('reset');
+    var cards = [];
+	var playingDecks = {
+		deckPlaying1: {array: [], cardPos: 1, click: checkCard},
```

Comment:
Bad naming. Don't use common names like array, set, object, etc since it's confusing. `playingDecks[key].array.push` below looks soooo puzzling